src/pages/docs/argo-cd/live-object-status/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2025-03-28
+modDate: 2025-05-16
+navSection: Live Object Status
+title: Argo CD Live Object Status
+navTitle: Live Object Status
+description: Argo CD Live Object Status guide.
+navOrder: 45
+hideInThisSectionHeader: true
+---
+Argo Live Object Status shows the live status of the Kubernetes resources deployed by the Argo CD Applications mapped
+to the project.
+This allows you to monitor and safely troubleshoot Argo CD controlled deployments from within Octopus Deploy
+
+:::div{.info}
+Outwardly this is an identical capability to that available for [Kubernetes based projects](/docs/kubernet/live-object-status/index.md).
+However when used with Argo CD, the [Kubernetes monitor](/docs/kubernetes/targets/kubernetes-agent/kubernetes-monitor) is *not* required.
```

Comment:
Most places we say `Kubernetes Agent` - so maybe say that here instead of the monitor